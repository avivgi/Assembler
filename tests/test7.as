.define sz = 2
K: .data 22
LIST: .data 6, 8, sz

; zero operands
rts

END: hlt