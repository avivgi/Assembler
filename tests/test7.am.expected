mov r2, r1, r3
